framework/xmlparser.js

```diff
@@ -49,13 +50,36 @@ function parse(xml, opts) {
 
 	// strip comments and whitespaces
 	xml = xml.trim();
-	xml = xml.replace(/<!--[\s\S]*?-->/g, '').replace(new RegExp("\\n" ,"g"), '<br />');
+	xml = xml.replace(/<!--[\s\S]*?-->/g, '').replace(new RegExp("\n" ,"g"), '');
+
+	// apply replacers
+	_.each(replacers, function(replacer, type) {
+		if (replacer.openTag == null) {
+			Ti.API.error('XMLParser: a replacer has been defined for', type, 'but with no openTag attribute. Defaulting to empty string.');
+			replacer.openTag = '';
+		}
+		if (replacer.closeTag == null) {
+			Ti.API.error('XMLParser: a replacer has been defined for', type, 'but with no closeTag attribute. Defaulting to empty string.');
+			replacer.closeTag = '';
+		}
+
```

Comment:
Since you're setting a default value, In my opinion not providing an `openTag` or `closeTag` shouldn't return an error but a warning

Reply:
I changed it to `warn`.